thirdeye/thirdeye-frontend/app/pods/custom/metrics-table-current/template.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="metrics-table__change">
+  &nbsp;
+</p>
+<p class="metrics-table__value">
+  {{get record (concat column.propertyName ".value")}}
+</p>
```

Comment:
When does this custom template get used? It's not clear to me.

Reply:
for the 'current' offset

Reply:
Since this template doesn't differ much from the `metrics-table-offset` template, can we do a check for whether it's a `current` column via the `propertyName` of the table and use 1 custom template for both current and offset columns?

Reply:
hmm. a state-check in a template? it's kind of a different thing - only displaying a value rather than value & change.